frontends/web/src/routes/account/send/send.tsx

```diff
@@ -59,7 +58,7 @@ interface SignProgress {
 type Props = SendProps & TranslateProps;
 
 interface State {
-    account?: Account;
+    account?: accountApi.IAccount;
```

Comment:
i am confused, this change is already in staging-aopp hmm

Reply:
?

I see `Account` https://github.com/digitalbitbox/bitbox-wallet-app/blob/staging-aopp/frontends/web/src/routes/account/send/send.tsx#L62

Reply:
Ah right, I was looking at the wrong line, sorry. Where was `Account` defined? I must be blind.

Reply:
you probably mean this here in the props interface https://github.com/digitalbitbox/bitbox-wallet-app/pull/1373/files#diff-b78d1ac6efeedc528ab327396d435b97de9497cd9354a099a1517cf50fc076ceR47